src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        String randomColor = ColorSupplier.getRandomColor();
+        int randomNumber = new Random().nextInt(100) + 1;
```

Comment:
1)the same here about random
2) magic numbers

Reply:
not fixed